data/uhc_pack/functions/tick.mcfunction

```diff
@@ -6,7 +6,7 @@
 # 4 - UHC
 # 5 - Awards
 
-execute if score UHC uhcState matches 1 run function uhc_pack:lobby/tick
+execute if score UHC uhcState matches 1..2 run function uhc_pack:lobby/tick
```

Comment:
Is this needed for the PR? If possible, It would be good to leave this out. I am happy to consider readdressing the states separately, but I'm not a fan of this change as it devalues the meaning of what uhcState means

Reply:
Ah yes, it can be left out for the PR :)
This was just a quick way to get around the issue

Reply:
I have raised #36 as a follow up action item.